cypress/integration/adminFlows/config/authenticationSection.spec.js

```diff
@@ -23,9 +23,11 @@ describe('Authentication Section', () => {
           cy.visit('/admin/customization/config');
           cy.findByTestId('authSectionForm').as('authSectionForm');
 
-          cy.get('@authSectionForm').findByText('Authentication').click();
           cy.get('@authSectionForm')
-            .findByLabelText('Invite-only mode')
+            .findByRole('heading', { name: 'Authentication' })
```

Comment:
This spec kept failing both on Travis and locally with:
```
  Running:  adminFlows/config/authenticationSection.spec.js                                (4 of 53)
  Authentication Section
    invite-only mode setting
      1) should disable email registration and all authorization providers when enabled
    authentication providers settings
      ✓ should display warning modal if provider keys are missing (3039ms)
      ✓ should not display warning modal if provider keys present (4214ms)
  2 passing (16s)
  1 failing
  1) Authentication Section
       invite-only mode setting
         should disable email registration and all authorization providers when enabled:
     AssertionError: Timed out retrying after 4000ms: expected '<input#settings_authentication_invite_only_mode.crayons-checkbox>' to be 'checked'
      at Context.eval **(http://127.0.0.1:39305/__cypress/tests?p=cypress/integration/adminFlows/config/authenticationSection.spec.js:130:74)**
```
I have a hunch that it's because I made changes to the `self.private?` and `self.invitation_only?`. The changes I made [here](https://github.com/forem/forem/pull/14161/files#diff-962d0ba30167fb8b3e14f5b7b485d768cf17aaa7d72630ec98a665989eaaa12dR49) and [here](https://github.com/forem/forem/pull/14161/files#diff-43838b45549995b359c4f90a902673ae8c84e758e671342ab671fed67f298292R11) help move the spec along, but it doesn't seem like it's quite there yet and I wonder if I'm going down the right rabbit hole by making all of these changes. 🤔